message_definitions/v1.0/common.xml

```diff
@@ -4766,6 +4766,16 @@
       <field type="uint8_t" name="breach_mitigation" enum="FENCE_MITIGATE">Active action to prevent fence breach</field>
     </message>
     <!-- MESSAGE IDs 180 - 229: Space for custom messages in individual projectname_messages.xml files -->
+    <message id="227" name="HYBRID_READ">
+      <description>Reading from a generator or other hybrid technology.</description>
+      <field type="uint32_t" name="run_time">Description.</field>
+      <field type="uint32_t" name="next_time">Description.</field>
+      <field type="uint16_t" name="run_status">Description.</field>
+      <field type="uint16_t" name="rate">Description.</field>
+      <field type="uint16_t" name="current">Description.</field>
```

Comment:
The BATTERY_STATUS and SYS_STATUS messages have current fields which are "int16_t" with units "cA".  We should probably be consistent with these other messages.